app/screens/login_options/login_options.js

```diff
@@ -209,6 +262,37 @@ export default class LoginOptions extends PureComponent {
         return null;
     };
 
+    renderOpenIdOption = () => {
+        const {config, license} = this.props;
+        const openIdEnabled = config.EnableSignUpWithOpenId === 'true' && license.IsLicensed === 'true' && isMinimumServerVersion(config.Version, 5, 32, 0);
```

Comment:
Sorry if this was already answered and I missed it somewhere, but regarding cloud, how will the minimum server version be handled?